### PR TITLE
Add legacy option

### PR DESCRIPTION
**Pull request by ThoumyreStanislas**
Hello, I'm a PhD student supervised by Professor Francois Rheault, and I've taken the liberty of modifying your code following errors reading vtk files in MI-Brain due to the version change between vtk5 and vtk4.2. https://discourse.vtk.org/t/can-we-write-out-the-old-vtk-4-2-file-format-with-vtk-9/5066
I therefore made sure that save_polydata saved the data in vtk4.2. 
@frheault

### CODE REVIEW

**Comment by frheault** [comment]
@StongeEtienne any chance you can merge that and release?

File: trimeshpy/vtk_util.py [hunk 1]
```diff
@@ -150,7 +150,8 @@ def save_polydata(polydata, file_name, binary=False, color_array_name=None):
     writer = set_input(writer, polydata)
     if color_array_name is not None:
         writer.SetArrayName(color_array_name)
-
+    if legacy:
+        writer.SetFileVersion(42)
```
Comment (StongeEtienne) [inline trimeshpy/vtk_util.py:153]: Change the name to "legacy_vtk_format"

File: trimeshpy/vtk_util.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def load_polydata(file_name):
 
 
 # Save
-def save_polydata(polydata, file_name, binary=False, color_array_name=None):
+def save_polydata(polydata, file_name, binary=False, color_array_name=None, legacy=True):
```
Comment (StongeEtienne) [inline trimeshpy/vtk_util.py]: Change the name to "legacy_vtk_format" and the default setting to  False

**Review by StongeEtienne — COMMENTED** [review]
Change the name to "legacy_vtk_format" and the default setting to False

**Comment by ThoumyreStanislas** [comment]
Hello, I've updated the script with the changes you wanted.

**Pull request merged**